tests/suit/ovirt_test.go

```diff
@@ -13,77 +14,159 @@ import (
+			err = utils.CreateProviderFromDefinition(f.CrClient, targetPr)
+			Expect(err).ToNot(HaveOccurred())
+			_, err = utils.WaitForProviderReadyWithTimeout(f.CrClient, namespace, utils.TargetProviderName, 30*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Create oVirt provider")
+			pr := utils.NewProvider(ovirtProviderName, forkliftv1.OVirt, namespace, map[string]string{}, map[string]string{}, f.OvirtClient.OvirtURL, s)
+			err = utils.CreateProviderFromDefinition(f.CrClient, pr)
+			Expect(err).ToNot(HaveOccurred())
+			provider, err := utils.WaitForProviderReadyWithTimeout(f.CrClient, namespace, ovirtProviderName, 30*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Create Network Map")
+			networkMapDef := utils.NewNetworkMap(namespace, *provider, networkMapName, vmData.GetVMNics()[0])
+			err = utils.CreateNetworkMapFromDefinition(f.CrClient, networkMapDef)
+			Expect(err).ToNot(HaveOccurred())
+			err = utils.WaitForNetworkMapReadyWithTimeout(f.CrClient, namespace, networkMapName, 10*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Create Storage Map")
+			storageMapDef := utils.NewStorageMap(namespace, *provider, test_storage_map_name, vmData.GetVMSDs(), ovirtStorageClass)
+			err = utils.CreateStorageMapFromDefinition(f.CrClient, storageMapDef)
+			Expect(err).ToNot(HaveOccurred())
+			err = utils.WaitForStorageMapReadyWithTimeout(f.CrClient, namespace, test_storage_map_name, 10*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Creating plan")
+			planDenf := utils.NewPlanWithVmId(*provider, namespace, test_plan_name, test_storage_map_name, networkMapName, "default", []string{vmData.GetTestVMId()})
+			err = utils.CreatePlanFromDefinition(f.CrClient, planDenf)
+			Expect(err).ToNot(HaveOccurred())
+			err, _ = utils.WaitForPlanReadyWithTimeout(f.CrClient, namespace, test_plan_name, 15*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Creating migration")
+			migrationDef := utils.NewMigration(provider.Namespace, test_migration_name, test_plan_name)
+			err = utils.CreateMigrationFromDefinition(f.CrClient, migrationDef)
+			Expect(err).ToNot(HaveOccurred())
+			err = utils.WaitForMigrationSucceededWithTimeout(f.CrClient, provider.Namespace, test_migration_name, 300*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+			By("Verifying imported VM exists")
+			vmId := types.UID(vmData.GetTestVMId())
+			vm, err := utils.GetImportedVm(f.CrClient, "default", func(vm cnv.VirtualMachine) bool {
+				return vm.Spec.Template.Spec.Domain.Firmware.UUID == vmId
+			})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(vm).ToNot(BeNil())
+		})
+
+		It("[oVirt MTV] should create insecure provider", func() {
+			namespace := f.Namespace.Name
+			By("Load Source VM Details from Ovirt")
+			vmData, err := f.Clients.OvirtClient.LoadSourceDetails()
+			Expect(err).ToNot(HaveOccurred())
+			By("Reset fakeovirt")
```

Comment:
i'm not sure how many tests(vms) we are going to add here, but maybe it worth to pull it out to `AfterEach`?

Reply:
good suggestion, although its required only for tests that using image transfer, beside the regular and insecure flow I don't know if we will have something soon, we can change it later if will be needed.